### PR TITLE
feat(ci): update release workflow to bump homebrew-core formula

### PR DESCRIPTION
**Pull request by nishantwrp**
**Description**

This PR is related to https://github.com/meshery/homebrew-tap/issues/10. 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/build-and-release-stable.yml [hunk 1]
```diff
@@ -59,9 +59,15 @@ jobs:
         with:
           version: latest
           args: release --rm-dist --skip-validate
+      - name: bump homebrew-core formula
+        uses: mislav/bump-homebrew-formula-action@v2
+        with:
+          formula-name: mesheryctl
+          download-url: https://github.com/meshery/meshery.git
```
Comment (leecalcote) [inline .github/workflows/build-and-release-stable.yml:66]: Update need to specify new location and release tag.

File: .github/workflows/build-and-release-stable.yml [hunk 2]
```diff
@@ -59,9 +59,15 @@ jobs:
...
+        env:
+          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_COMMITTER_TOKEN }}
```
Comment (leecalcote) [inline .github/workflows/build-and-release-stable.yml:68]:
```suggestion
          COMMITTER_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```

**Comment by leecalcote** [comment]
It's good to see this, @nishantwrp 👍

File: .github/workflows/build-and-release-stable.yml (hunk 2 above)
Comment (nishantwrp) [reply .github/workflows/build-and-release-stable.yml]: Just want to confirm that it has `public_repo` access.

File: .github/workflows/build-and-release-stable.yml (hunk 1 above)
Comment (nishantwrp) [reply .github/workflows/build-and-release-stable.yml:66]: I'm sorry didn't understand your question. Are you asking if we need to explicitly configure new tap location and release tag? 

If yes, this gh action assumes the location to be `homebrew-core` if not specified, tag defaults to the currently pushed tag if not specified.

**Pull request merged**